python/sglang/srt/layers/moe/moe_runner/runner.py

```diff
@@ -11,6 +11,7 @@
 )
 from sglang.srt.layers.moe.moe_runner.deep_gemm import DeepGemmRunnerCore
 from sglang.srt.layers.moe.moe_runner.triton import TritonRunnerCore
+from sglang.srt.layers.moe.moe_runner.triton_kernels import TritonKernelsRunnerCore
```

Comment:
I find that we mixed the usage of `triton_kernels` and `triton_kernel`. We'd better keep their usage consistent.